Source/Core/DolphinQt2/GameList/GameTracker.h

```diff
@@ -39,7 +40,7 @@ class GameTracker final : public QFileSystemWatcher
 
 signals:
   void GameLoaded(std::shared_ptr<const UICommon::GameFile> game);
-  void GameRemoved(const QString& path);
+  void GameRemoved(const std::string& path);
```

Comment:
``std::string`` will need to be declared as a metatype (``Q_DECLARE_METATYPE``) if it isn't somewhere else already.

Reply:
Done.